APIManager.js

```diff
@@ -0,0 +1,91 @@
+//This is the class that will manage all your APIs
+
+class APIManager {
+    constructor() {
+        this.data = {
+            user: {},
+            friends: [],
+            quote: '',
+            pokemon: {},
+            about: ''
+        }
+    }
+
+    getUser() {
+        $.ajax({
+            method: 'GET',
+            url: 'https://randomuser.me/api/',
+            success:  (users) => {
```

Comment:
This is kind of repetative. Try something like:
`const person = users.results[0]`

Then:

`this.data.user = { fName: person.name.first, lName: person.name.last, .... }`

It will be less of the same exact words.